src/ProjectTemplates/Web.ProjectTemplates/content/Components-CSharp/_Imports.razor

```diff
@@ -0,0 +1,9 @@
+﻿@using System.Net.Http
+@using System.Net.Http.Json
+@using Microsoft.AspNetCore.Components.Forms
+@using Microsoft.AspNetCore.Components.Routing
```

Comment:
```suggestion
@using Microsoft.AspNetCore.Components.Routing
@using Microsoft.AspNetCore.Components.Sections
```

Reply:
QuickGrid is a separate package, but sections are included in the framework just like the forms components, right?